page-hope.hbs

```diff
@@ -1,7 +1,7 @@
 {{!< decalDefault}}
 <main>
 	{{#post}}
-		<article style="width: 90vw;">
+		<article style="width: 1000px;">
```

Comment:
Why are we using static values here?

Reply:
The webpage is constant for all laptop computers. Its better to have fixed pixels for this since u can just scroll left on mobile for the table

Reply:
Is it just the table that needs scrolling to the left?

Reply:
Mostly yeah. There is no reason to make the course website relative to display width

Reply:
It's just that limiting the width to `1000px` would affect users on computers that have a very wide screen. The purpose of `90vw` was to give the edges a margin.

Reply:
I created the 90vw in the first place because it was for the previous version of the website. The text is unreadable anyways if it gets too wide and 1000px worked just fine for all desktops

Reply:
Imagine the cs70 website